### PR TITLE
Cog 1393 update component library

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
[COG-1393](https://osi-cwds.atlassian.net/browse/COG-1393)

#### Brief feature description
-Updated to latest component library, button component changed as primitiveButton and checkbox component is added within the application since it is deprecated in the latest version of component library.

### CODE REVIEW

File: app/javascript/views/userDetail/ChangeLogDetailsView.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Modal, ModalBody, ModalHeader, Avatar, Button } from '@cwds/components'
+import { Modal, ModalBody, ModalHeader, Avatar, PrimitiveButton as Button } from '@cwds/components'
```
Comment (kelfink) [inline app/javascript/views/userDetail/ChangeLogDetailsView.jsx]: couldn't we just keep the PrimitiveButton name and rename the references in this file?  Seems like that is less confusing, for those who have/will work in both libraries.  Just an opinion about that, though.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
     "seed-users": "scripts/seed-users"
   },
   "dependencies": {
-    "@cwds/components": "0.6.0",
+    "@cwds/components": "1.2.1",
+    "@cwds/forms": "1.2.1",
```
Comment (kelfink) [inline package.json]: I think we no longer need the forms package, and probably shouldn't include it, since it's not ready for production

**Comment by Sushmareddy-OSI** [comment]
retest this please

File: app/javascript/views/userDetail/ChangeLogDetailsView.jsx (hunk 1 above)
Comment (Sushmareddy-OSI) [reply app/javascript/views/userDetail/ChangeLogDetailsView.jsx]: If we reference as PrimitiveButton in each file, that will be tough to change in every place when it changes again.

Comment (kelfink) [reply app/javascript/views/userDetail/ChangeLogDetailsView.jsx]: You are so pessimistic!  And probably right.  OK

**Pull request merged**